src/providers/trusted_service/context/asym_encryption.rs

```diff
@@ -6,22 +6,16 @@ use super::ts_protobuf::{
 use super::Context;
 use parsec_interface::operations::psa_algorithm::AsymmetricEncryption;
 use parsec_interface::requests::ResponseStatus;
-use std::convert::TryInto;
-use zeroize::Zeroize;
 
 impl Context {
     pub fn asym_encrypt(
         &self,
         key_id: u32,
         alg: AsymmetricEncryption,
-        mut plaintext: Vec<u8>,
```

Comment:
I am assuming zeroisation is no longer required?

Reply:
Hmm, what I find interesting is that we didn't zeroize it after sending the request, even though we don't pass ownership of the plaintext data, just a reference to it. Maybe we should do it before returning, regardless of whether it's an exception or not. Or maybe we didn't do it for some reason and I forgot the reason 🤔

Would it be possible for you to quickly check if zeroizing at the end breaks anything?

Reply:
I thought all the structs from ts_protobuf have Drop trail implemented [here](https://github.com/parallaxsecond/parsec/blob/main/src/providers/trusted_service/context/ts_protobuf.rs#L62). But I can only see Drop being implemented for `ImportKeyIn`, `ExportKeyOut`, `SignHashIn`, `VerifyHashIn`. For these structs the drop method takes care of the zeroise part. 

Not sure why `AsymmetricEncryptIn` and `AsymmetricDecryptIn` don't have them.

Reply:
The test [patch](https://github.com/parallaxsecond/parsec/pull/747/commits/87b6e7206ab9212aaaa287ff72ee82191ce1a489) seems to work fine and there are no failures reported in the CI jobs for trusted services backend.

Reply:
I would be more inclined for the Drop method of the involved types to handle the zeroization. Could you check if this is possible and easy to do? I'm approving in case it's not possible/easy.